stripe/src/main/java/com/stripe/android/view/CardInputWidget.java

```diff
@@ -680,7 +657,7 @@ protected void applyTransformation(float interpolatedTime, Transformation t) {
 
         final int dateDestination =
                 mPlacementParameters.peekCardWidth
-                + mPlacementParameters.cardDateSeparation;
```

Comment:
I suspect an Android Studio update reformatted things when I touched this class. I thought it best not to fight, since I needed to add tests anyhow.